lib/DDGC/Web/Controller/InstantAnswer.pm

```diff
@@ -848,34 +846,50 @@ sub create_ia :Chained('base') :PathPart('create') :Args() {
         if ($is_admin) {
             my $dev_milestone = $c->req->params->{dev_milestone};
             my $status = $dev_milestone;
-            
-            if ($dev_milestone eq 'development') {
-                $status =~ s/_/ /g;
-            }
 
-            my $new_ia = $c->d->rs('InstantAnswer')->create({
-                id => lc($c->req->params->{id}),
-                meta_id => lc($c->req->params->{id}),
-                name => $c->req->params->{name},
-                status => $status,
-                dev_milestone => $dev_milestone,
-                description => $c->req->params->{description},
-            });
+            $id = format_id($c->req->params->{id});
+           
+            if (length $id) { 
+                my $new_ia = $c->d->rs('InstantAnswer')->create({
+                    id => $id,
+                    meta_id => $id,
+                    name => $c->req->params->{name},
+                    status => $status,
+                    dev_milestone => $dev_milestone,
+                    description => $c->req->params->{description},
+                });
 
-            save_milestone_date($new_ia, 'created');
+                save_milestone_date($new_ia, 'created');
 
-            $result = 1;
+                $result = 1;
+            }
         }
     }
 
     $c->stash->{x} = {
         result => $result,
+        id => $id
     };
 
     $c->stash->{not_last_url} = 1;
     return $c->forward($c->view('JSON'));
 }
 
+sub format_id {
+    my( $id ) = @_;
+
+    # id must be unique, lowercase and without weird chars
+    $id =~ s/[^a-zA-Z0-9]+/_/g;
+    $id =~ s/^[^a-zA-Z]//;
+    $id =~ s/[^a-zA-Z0-9]$//;
```

Comment:
Yeah I just want to totally remove them at the end (because the first line replaces all of them with underscores, so if there's something at the end of the string we still need to remove that last underscore)